src/libraries/System.Private.CoreLib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -681,6 +681,187 @@ private bool TryInsert(TKey key, TValue value, InsertionBehavior behavior)
             return true;
         }
 
+        /// <summary>
```

Comment:
Since this is essentially duplicating the logic of `Dictionary.TryInsert`, could you add a comment in both methods advising maintainers to keep them in sync?

Reply:
Done in 5209a5bedff52a33454da401854502d6b967501a 🙂